### DISCUSSION TITLE
Working with list menus / dropdown menu

### DISCUSSION THREAD

**Discussion by yuirriiu** [jef/streetmerchant#1841] [Help]
Gday,
is there a way of working with selection menus for different product versions ? For example: imagine the static webshop url "www.shop.com/ps5", there you need to choose via a selection menu, if you wanna have the digital oder non digital version. Is there a way for only scanning for one version of the prodcut ? Can't figure out a way to do that via model file.
Thanks in advance

**Comment by wofnull** [comment]
in most of the times, the page has an unique URL for different versions, even if a drop own or klickable selection is exisiting ( like on amazon ). However this is most of the times not reflected in URL of the site itself. You can find the correct URL to the different versions most likely in the website Source Code ;)
